cardano-node/src/Cardano/Node/Configuration/Logging.hs

```diff
@@ -24,10 +24,11 @@ module Cardano.Node.Configuration.Logging
 
 import           Cardano.Prelude hiding (trace)
 
+import qualified Control.Concurrent as Conc
 import qualified Control.Concurrent.Async as Async
 import           Control.Exception.Safe (MonadCatch)
 import           Control.Monad.Trans.Except.Extra (catchIOExceptT)
-import           Control.Tracer
+import           "contra-tracer" Control.Tracer
```

Comment:
As is written in the README of trace-dispatcher: "We integrated contra-tracer-0.1.0.0 into the source tree, because the iohk-monitoring framework depends on the non-arrow based contra-tracer framework.
This should become a dependency when old tracing gets removed, and it is a temporary hack." But for this reason we obviously don't want to change anything in the code of the copy.

Reply:
Just to clarify the `Control.Tracer` module that sits in `trace-dispatcher` will be removed when old tracing gets removed?

Reply:
Yes exactly